crates/adapterlib/src/transport.rs

```diff
@@ -318,6 +356,25 @@ impl<A> InputQueue<A> {
         }
     }
 
+    /// Appends `buffer`, to the queue, and associates it with `aux`.  Reports
+    /// to the controller that `errors` have occurred during parsing.
+    ///
+    /// If `buffer` is empty, then this discards `aux`, even if `buffer` is
```

Comment:
I don't see this in the code, where does it happen?

Reply:
right, this comment is wrong